docs/UserGuide.md

```diff
@@ -545,13 +538,15 @@ Parameter | Prefix | Constraints, Examples
 **INDEX** | - | Index is a number greater than 0 that is based on the numberings beside each client or note. <br> e.g. `1` would refer to the first client or note.
 
 --------------------------------------------------------------------------------------------------------------------
+
 ## Appendix A - Introduction to the UI
 
 ### Start Page
 
 The diagram below shows the key UI elements of TBM upon first start-up.
 
-![Annotated UI Default Page](images/command-screenshots/UI_annotated.png)
+  ![Annotated UI Default Page](images/command-screenshots/UI_annotated.png)
+  Figure 2. Default View
```

Comment:
Should all the figures be wrapped in <p align="center"> ? Also, maybe make them italicised

Reply:
I think it is auto centered in gh-pages